opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -777,3 +782,27 @@ def test_os_detector_solaris(self):
 
         self.assertEqual(resource.attributes[OS_TYPE], "solaris")
         self.assertEqual(resource.attributes[OS_VERSION], "666.4.0.15.0")
+class TestHostResourceDetector(unittest.TestCase):
+    def test_host_resource_detector(self):
+        detector = HostResourceDetector()
+        actual = detector.detect()
+        self.assertTrue(actual.attributes.get(ResourceAttributes.HOST_NAME))
+        self.assertTrue(actual.attributes.get(ResourceAttributes.HOST_ARCH))
+
+    def test_host_resource_as_span_attribute(self):
```

Comment:
This test is fine, but does it add coverage to the changes submitted by this PR?

Reply:
I was expecting a more comprehensive test where we can assert the actual values of the attributes `host.arch` and `host.name`. For example, the test could check specific values like:

host.arch == "arm64"
host.name == "opentelemetry-test"

This would ensure that the HostResourceDetector detects the attributes and populates them correctly with the expected values.